csharp/ql/test/library-tests/dataflow/modulusanalysis/ModulusAnalysis.expected

```diff
@@ -0,0 +1,71 @@
+| ModulusAnalysis.cs:21:15:21:23 | ... + ... | 0 | 3 | 4 |
+| ModulusAnalysis.cs:21:19:21:19 | 4 | 0 | 4 | 0 |
+| ModulusAnalysis.cs:21:23:21:23 | 3 | 0 | 3 | 0 |
+| ModulusAnalysis.cs:22:15:22:15 | access to parameter i | SSA param(i) | 0 | 0 |
+| ModulusAnalysis.cs:22:15:22:19 | ... * ... | 0 | 0 | 8 |
+| ModulusAnalysis.cs:22:15:22:23 | ... + ... | 0 | 7 | 8 |
+| ModulusAnalysis.cs:22:19:22:19 | 8 | 0 | 8 | 0 |
+| ModulusAnalysis.cs:22:23:22:23 | 7 | 0 | 7 | 0 |
+| ModulusAnalysis.cs:23:34:23:34 | access to local variable j | 0 | 3 | 4 |
+| ModulusAnalysis.cs:23:34:23:34 | access to local variable j | SSA def(j) | 0 | 0 |
+| ModulusAnalysis.cs:25:13:25:13 | access to parameter x | SSA param(x) | 0 | 0 |
+| ModulusAnalysis.cs:25:17:25:18 | access to constant c1 | 0 | 42 | 0 |
+| ModulusAnalysis.cs:25:17:25:18 | access to constant c1 | SSA entry def(ModulusAnalysis.c1) | 0 | 0 |
+| ModulusAnalysis.cs:25:23:25:23 | 3 | 0 | 3 | 0 |
+| ModulusAnalysis.cs:25:28:25:28 | access to parameter y | SSA param(y) | 0 | 0 |
+| ModulusAnalysis.cs:25:32:25:33 | access to constant c1 | 0 | 42 | 0 |
+| ModulusAnalysis.cs:25:32:25:33 | access to constant c1 | SSA entry def(ModulusAnalysis.c1) | 0 | 0 |
+| ModulusAnalysis.cs:25:38:25:38 | 7 | 0 | 7 | 0 |
+| ModulusAnalysis.cs:27:38:27:38 | access to parameter x | 0 | 3 | 42 |
+| ModulusAnalysis.cs:27:38:27:38 | access to parameter x | SSA param(x) | 0 | 0 |
+| ModulusAnalysis.cs:27:38:27:42 | ... + ... | 0 | 10 | 42 |
+| ModulusAnalysis.cs:27:38:27:42 | ... + ... | SSA param(x) | 7 | 42 |
+| ModulusAnalysis.cs:27:38:27:42 | ... + ... | SSA param(y) | 3 | 42 |
+| ModulusAnalysis.cs:27:42:27:42 | access to parameter y | 0 | 7 | 42 |
+| ModulusAnalysis.cs:27:42:27:42 | access to parameter y | SSA param(y) | 0 | 0 |
+| ModulusAnalysis.cs:30:13:30:13 | access to parameter x | SSA param(x) | 0 | 0 |
+| ModulusAnalysis.cs:30:17:30:18 | access to constant c1 | 0 | 42 | 0 |
+| ModulusAnalysis.cs:30:17:30:18 | access to constant c1 | SSA entry def(ModulusAnalysis.c1) | 0 | 0 |
+| ModulusAnalysis.cs:30:23:30:23 | 3 | 0 | 3 | 0 |
+| ModulusAnalysis.cs:30:28:30:28 | access to parameter y | SSA param(y) | 0 | 0 |
+| ModulusAnalysis.cs:30:32:30:33 | access to constant c1 | 0 | 42 | 0 |
+| ModulusAnalysis.cs:30:32:30:33 | access to constant c1 | SSA entry def(ModulusAnalysis.c1) | 0 | 0 |
+| ModulusAnalysis.cs:30:38:30:38 | 7 | 0 | 7 | 0 |
+| ModulusAnalysis.cs:32:38:32:38 | access to parameter x | 0 | 3 | 42 |
+| ModulusAnalysis.cs:32:38:32:38 | access to parameter x | SSA param(x) | 0 | 0 |
+| ModulusAnalysis.cs:32:38:32:42 | ... - ... | 0 | 38 | 42 |
+| ModulusAnalysis.cs:32:38:32:42 | ... - ... | SSA param(x) | 35 | 42 |
+| ModulusAnalysis.cs:32:42:32:42 | access to parameter y | 0 | 7 | 42 |
+| ModulusAnalysis.cs:32:42:32:42 | access to parameter y | SSA param(y) | 0 | 0 |
+| ModulusAnalysis.cs:35:17:35:26 | access to property Length | SSA untracked def(arr.Length) | 0 | 0 |
```

Comment:
I expected this to be an `ExprBound` and not an `SsaBound`.
Looks like we get an `ImplicitUntrackedDefinition`, which might be okay, but we should also get the `ExprBound`. I'll check what happens in Java.